### PR TITLE
Added document loaders for eml and msg files.

### PR DESCRIPTION
**Pull request by pgebert**
Added additional document loaders to index mails (eml and msg file types).

Closes (partially) #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ If you want to start from an empty database, delete the `db` folder.
 
 Note: during the ingest process no data leaves your local environment. You could ingest without an internet connection.
 
+Supported file types:
```
Comment (imartinez) [inline README.md:49]: Could you move this to the beginning of the "Instructions for ingesting your own dataset" section? And probably modify the first phrase where it says "put any and all of your .txt, .pdf or .csv files..." so the whole section is aligned. 

Id suggest:

Put any and all of your files into the source_documents directory

Supported file types:
- txt
...

**Review by imartinez — CHANGES_REQUESTED** [review]
Nice addition! Please take a look at my only comment. With that it'll be ready to merge

File: ingest.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from typing import List
 from dotenv import load_dotenv
 
-from langchain.document_loaders import TextLoader, PDFMinerLoader, CSVLoader
+from langchain.document_loaders import TextLoader, PDFMinerLoader, CSVLoader, UnstructuredEmailLoader, OutlookMessageLoader
```
Comment (imartinez) [inline ingest.py:6]: After reviewing LangChain documentation for the two loaders, I think you are missing new requirements to be installed, that should be added to the requirements file:

unstructured
extract_msg

**Review by imartinez — CHANGES_REQUESTED** [review]

File: ingest.py (hunk 2 above)
Comment (pgebert) [reply ingest.py:6]: Oh sorry. Added the two dependencies to the `requirements.txt`.

File: README.md (hunk 1 above)
Comment (pgebert) [reply README.md]: Done

**Comment by imartinez** [comment]
@pgebert what do you think about #211 ? I think it covers the same functionality and the approach is a little more extensible from a code perspective. If you agree we could close this PR and go with #211 .

**Comment by imartinez** [comment]
Finally merged #211. Thanks a lot for your help! Closing this PR, feel free to open a new one extending the documents support.

**Pull request closed without merging**